### PR TITLE
Add FACTCK.BR dataset

### PR DESCRIPTION
**Pull request by hugoabonizio**
This PR adds [FACTCK.BR](https://github.com/jghm-f/FACTCK.BR) dataset from [FACTCK.BR: a new dataset to study fake news](https://dl.acm.org/doi/10.1145/3323503.3361698).

### CODE REVIEW

**Review by lhoestq — COMMENTED** [review]
Nice thanks !

I left a comment about the feature type of the label field

File: datasets/factckbr/factckbr.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                {
+                    "url": datasets.Value("string"),
+                    "author": datasets.Value("string"),
+                    "date": datasets.Value("string"),
+                    "claim": datasets.Value("string"),
+                    "review": datasets.Value("string"),
+                    "title": datasets.Value("string"),
+                    "rating": datasets.Value("float"),
+                    "best_rating": datasets.Value("float"),
+                    "label": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/factckbr/factckbr.py:75]: Can you use the ClassLabel feature type here ?
For example
```
"label": datasets.ClassLabel(names=["falso", etc.])
```

**Comment by hugoabonizio** [comment]
@lhoestq done!

File: datasets/factckbr/factckbr.py [hunk 2]
```diff
@@ -103,6 +120,11 @@ def _generate_examples(self, filepath):
                 if id_ == 0:
                     continue
 
+                label = row[8].lower()
+
+                if label == "":
+                    continue
```
Comment (lhoestq) [inline datasets/factckbr/factckbr.py:126]: Are there rows with missing labels ?

In this case we should keep them and replace the missing label value with `-1`

The user is free to remove the bad rows using `.filter` for example

**Review by lhoestq — COMMENTED** [review]
Thanks for doing this change !

I have one comment about the missing labels though

File: datasets/factckbr/factckbr.py (hunk 2 above)
Comment (hugoabonizio) [reply datasets/factckbr/factckbr.py]: There are 4 examples with missing labels, but I wasn't aware of this convention. Sorry!

**Review by lhoestq — APPROVED** [review]
Thank you !

Looks all good now :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**